hdw.dat.sas

```diff
@@ -11,6 +11,10 @@
 # UNTIL 10/Dec/1996 17:00:00.0
     5 1996 29782800   52.160 -106.530  494.0   23.1  3.24  1   10 -0.158  1    0.0 -100.0    0.0 100.0  2  75 16
 #
+# UNTIL 20/Dec/2018 16:11:00.0 When Borealis started
+    5 2018 30613740   52.160 -106.530  494.0   23.1  3.24  1   10  0.000  1    0.0 -100.0    0.0 0.0  0  225 16
```

Comment:
I think this `yrsec` corresponds to `20181221 07:49:00 UT`, rather than `20181220 16:11:00 UT` as the comment line indicates.  Should the `yrsec` instead be `30557460`?